### DISCUSSION TITLE
Shape of solution outputs

### DISCUSSION THREAD

**Discussion by Walser52** [pybamm-team/PyBaMM#3606] [Q&A]
I took a simple code from the documentation.

```
import pybamm as pb
import numpy as np
import matplotlib.pyplot as plt

experiment = pb.Experiment(
    [
        (
            "Discharge at C/10 for 10 hours or until 3.3 V",
            "Rest for 1 hour",
            "Charge at 1 A until 4.1 V",
            "Hold at 4.1 V until 50 mA",
            "Rest for 1 hour",
        )
    ]
    * 3,
)
model = pb.lithium_ion.DFN()
sim = pb.Simulation(model, experiment=experiment, solver=pb.CasadiSolver())
sim.solve()

print(sim.solution.data['Negative particle surface concentration [mol.m-3]'].shape)
print(sim.solution.get_data_dict()['Voltage [V]'].shape)
```

It runs fine except that I am not sure why the shape output by the first print statement is (20, 2353) but the second one is (2353,). Could someone explain the 20 and what do each of the rows correspond to?

**Comment by valentinsulzer** [comment]
'Negative particle surface concentration [mol.m-3]' depends on both position in the electrode (closer to current collector or closer to separator) and time, the 20 rows are different positions in the electrode

**Reply by Walser52** [reply]
That's what I guessed but I couldn't find anything about how they're spaced or whether they're in the direction perpendicular to the electrode plane or in the plane. 

How do I find the coordinates of each measurement? Thank you.
